OneSignalSDK/onesignal/src/main/java/com/onesignal/OSSystemConditionController.java

```diff
@@ -30,9 +30,13 @@ boolean systemConditionsAvailable() {
             return false;
         }
 
-        if (isDialogFragmentShowing(ActivityLifecycleHandler.curActivity)) {
-            OneSignal.onesignalLog(OneSignal.LOG_LEVEL.WARN, "OSSystemConditionObserver dialog fragment detected");
-            return false;
+        try {
+            if (isDialogFragmentShowing(ActivityLifecycleHandler.curActivity)) {
+                OneSignal.onesignalLog(OneSignal.LOG_LEVEL.WARN, "OSSystemConditionObserver dialog fragment detected");
+                return false;
+            }
+        } catch (NoClassDefFoundError exception) {
+            OneSignal.onesignalLog(OneSignal.LOG_LEVEL.ERROR, "isDialogFragmentShowing AppCompatActivity not imported either used: " + exception);
```

Comment:
This can be an info, as it it isn't an error if  they don't have that class in their app.

Reply:
@Jeasmine This line is still using `OneSignal.LOG_LEVEL.ERROR`. Should be INFO.